autogen/docs/dictionary.html.mustache

```diff
@@ -1007,9 +1007,9 @@ show approximate-hsb 180 57.143 76.863
         </p>
         <p>
           The color reported may be only an approximation, since the NetLogo
-          color space does not include all possible colors. (See <a href="#approximate-hsb">approximate-hsb</a> for a description of what
-          parts of the HSB color space NetLogo colors cover; this is
-          difficult to characterize in RGB terms.)
+          color space does not include all possible colors. (See the Programming Guide
+          information on approximate-hsb and approximate-rgb for an example that demonstrates how
+          these primitives are approximations).
```

Comment:
I don't see this example in the programming guide, just a quick note about the NetLogo color space not include all possible colors.  But if it does exist, or we're going to add it, or we're just talking about the existing text, then we should link directly to it from here.  [Example](https://github.com/NetLogo/NetLogo/blob/hexy/autogen/docs/dictionary.html.mustache#L847)

Reply:
I think the parenthetical remark should wait for Isaac's upcoming  PR for the Programming Guide, or he could add the link in the Prog Guide PR.